Web/HTML Assignment-1/Question-1.html

```diff
@@ -0,0 +1,176 @@
+<!DocType html>
+<html>
+	<head>
+		<Title>
```

Comment:
Always keep symmetry for html tags,all tags should be in similar case either uppercase or lowercase.